modules/client/test/unit/context-plugins.test.ts

```diff
@@ -27,29 +36,47 @@ describe("ContextPlugin instances", () => {
       web3Providers: web3endpoints.working,
       gasFeeEstimationFactor: 0.1,
       graphqlNodes: [],
+      ipfsNodes: [],
     };
   });
-  it("Should create an empty context", () => {
-    const context = new ContextPlugin({});
-
-    expect(context).toBeInstanceOf(ContextPlugin);
-  });
   it("Should create an empty context and have default values", () => {
-    const context = new ContextPlugin({});
+    const context = new ContextPlugin();
 
     expect(context).toBeInstanceOf(ContextPlugin);
-    expect(context.signer).toEqual(undefined);
-    expect(context.gasFeeEstimationFactor).toEqual(0.625);
+    expect(context.network.name).toBe("homestead");
+    expect(context.network.chainId).toBe(1);
+    expect(context.daoFactoryAddress).toBe(LIVE_CONTRACTS.homestead.daoFactory);
+    expect(context.ensRegistryAddress).toBe(context.network.ensAddress);
+    expect(context.gasFeeEstimationFactor).toBe(0.625);
+    expect(context.web3Providers.length).toBe(WEB3_NODES.homestead.length);
+    expect(context.ipfs.length).toBe(IPFS_NODES.homestead.length);
+    expect(context.graphql.length).toBe(GRAPHQL_NODES.homestead.length);
+    context.web3Providers.map((provider) => {
+      expect(provider).toBeInstanceOf(JsonRpcProvider);
+    });
+    context.ipfs.map((ipfsClient) => {
+      expect(ipfsClient).toBeInstanceOf(IpfsClient);
+    });
+    context.graphql.map((graphqlClient) =>
+      expect(graphqlClient).toBeInstanceOf(GraphQLClient)
+    );
   });
   it("Should create a context and have the correct values", () => {
     const context = new ContextPlugin(contextParams);
 
     expect(context).toBeInstanceOf(ContextPlugin);
-    expect(context.network).toEqual("mainnet");
+    expect(context.network.name).toEqual("homestead");
```

Comment:
It shouldnt break anything on the app becuse i believe they dont use the netowrk param of the context however, I'll mention it in the changelog just in case

Reply:
Is it done?

Reply:
yep 

- `Context.network` type changes from `Networkish` to `Network`